src/lib.rs

```diff
@@ -54,6 +59,8 @@ use table::BucketState::{
     Full,
 };
 
+pub use adaptive_map::HashMapInterface;
```

Comment:
Yes, temporarily. As long as specialization doesn't work for inherent impls, users of adaptive hashing must import this trait.